cancer_ai/base/neuron.py

```diff
@@ -117,7 +117,8 @@ def sync(self, retries=5, delay=10, force_sync=False):
             try: 
                 # Ensure miner or validator hotkey is still registered on the network.
                 self.check_registered()
-
+                if self.config.filesystem_evaluation:
+                        break
```

Comment:
not syncing metagraph on local evaluation? dont we use models from the real competitions for local evaluation as well? its sometimes useful in troubleshooting
maybe we should set yet another flag to enable/disable resync if it bothers you

Reply:
it's just  so logs don't go that it's trying to set weights all the time, corrupts the logs. local evaluation doesn't need to sync, it just needs to download miner models. 
it  doesn't affect downloading data from the chain